.github/workflows/build-checks.yml

```diff
@@ -81,6 +90,7 @@ jobs:
             -DMAD=ON \
             -DMODPLUG=ON \
             -DWAVPACK=ON \
+            -DCMAKE_CXX_FLAGS="-Wno-nan-infinity-disabled" \
```

Comment:
could you add a comment at the issue locations so we don't forget to remove this once fixed?